config/districts.yml

```diff
@@ -1,2 +1,30 @@
-- Eixample
-- Ciutat Vella
+- 
+  - Ciutat Vella
```

Comment:
Well yeah, basically for readability. In other words: What you did is uuuuugly :boar: